desktop/src/main/java/bisq/desktop/main/settings/preferences/PreferencesView.java

```diff
@@ -120,21 +119,19 @@
     private ComboBox<TradeCurrency> preferredTradeCurrencyComboBox;
 
     private ToggleButton showOwnOffersInOfferBook, useAnimations, useDarkMode, sortMarketCurrenciesNumerically,
-            avoidStandbyMode, useCustomFee, autoConfirmXmrToggle, hideNonAccountPaymentMethodsToggle, denyApiTakerToggle,
+            avoidStandbyMode, autoConfirmXmrToggle, hideNonAccountPaymentMethodsToggle, denyApiTakerToggle,
             notifyOnPreReleaseToggle;
     private int gridRow = 0;
     private int displayCurrenciesGridRowIndex = 0;
-    private InputTextField transactionFeeInputTextField, ignoreTradersListInputTextField, ignoreDustThresholdInputTextField,
+    private InputTextField ignoreTradersListInputTextField, ignoreDustThresholdInputTextField,
```

Comment:
Removing this field means you cannot set a manual fee for BSQ transactions.

Reply:
You are right @sqrrm. I did that as there is this issue #5304 that is similarly taking care of BSQ and that would be next to the merge of this one. However I could restore that line as you said if you think it is better that way.

Reply:
As it is not sure how quickly #5304 will be implemented I think it would be good to leave this duplication for now and remove it with the refactoring for BSQ. WDYT?

Reply:
Agree with you. Just pushed this change.

Reply:
You just restored the reference for the InputTextField, but still removed the creation and functionality of it in Preferences. What I meant was to restore the functionality as it was before, but extract the UI element creation and most of it's functionality to FormBuilder and GUIUtil to prevent unnecessary duplication. Afterwards most of it can be re-used or adapted for the BSQ custom fee setting as well and removed from Preferences completely.